libappstream-glib/as-node.c

```diff
@@ -844,6 +864,7 @@ as_node_from_xml_internal (const gchar *data, gssize data_sz,
 	helper.flags = flags;
 	helper.current = root;
 	helper.locales = g_get_language_names ();
+	helper.is_em_text = helper.is_code_text = 0;
```

Comment:
I think it would be better to prevent future incidents to just have `AsNodeToXmlHelper helper = {0}` everywhere it's allocated.

Reply:
Changed as requested.